### PR TITLE
Upd/Update SDK to RC.8

### LINKED ISSUE #2320: Upgrade to SDK RC8
Once it's out, upgrade to the latest version. Depends on #2319.

### PR DESCRIPTION
**Pull request by notimetoname**
Closes #2320. Not the best update i did. Please look at the code that uses _a little bit_ new common `pivot` when reporting load by SN.

### CODE REVIEW

File: cmd/neofs-node/container.go [hunk 1]
```diff
@@ -400,11 +401,13 @@ func (l *loadPlacementBuilder) BuildPlacement(epoch uint64, cnr cid.ID) ([][]net
 		return nil, err
 	}
 
-	const pivotPrefix = "load_announcement_"
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply cmd/neofs-node/container.go:403]: How will this affect the logic?

**Review by roman-khimov — COMMENTED** [review]
`object/relations`, `object/slicer`?

**Comment by roman-khimov** [comment]
> do you mean that and that?

Sure.

File: cmd/neofs-adm/internal/modules/morph/subnet.go [hunk 2]
```diff
@@ -146,7 +147,10 @@ var cmdSubnetCreate = &cobra.Command{
 
 		// declare creator ID and encode it
 		var creator user.ID
-		user.IDFromKey(&creator, key.PrivateKey.PublicKey)
+		err = user.IDFromSigner(&creator, neofsecdsa.SignerRFC6979(key.PrivateKey))
+		if err != nil {
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply cmd/neofs-adm/internal/modules/morph/subnet.go:151]: > i would add some changes to SDK

Issue number? It's not something set in stone (yet).

[the remark this answers is not in the document]
Comment (cthulhu-rider) [reply cmd/neofs-adm/internal/modules/morph/subnet.go:151]: no way, error must never be ignored. If u 100% expect it should never occur for ur input, then at least panic. Imagine you can't see `user.IDFromSigner` implementation: it does not promise success in docs, so you need to be prepared for failure

File: cmd/neofs-node/container.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (cthulhu-rider) [reply cmd/neofs-node/container.go:403]: i always thought that using placement functions outside object placement context is a bad idea

> different (broken) container estimations may appear

probably yes. That does't bother me, nodes may (and may not) report any estimations, they aren't checked. But to make things unbroken, I'd just directly use HRW sorting

File: cmd/neofs-adm/internal/modules/morph/subnet.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (notimetoname) [reply cmd/neofs-adm/internal/modules/morph/subnet.go:151]: > Issue number?

https://github.com/nspcc-dev/neofs-sdk-go/issues/416

File: cmd/neofs-node/container.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (notimetoname) [reply cmd/neofs-node/container.go:403]: so i would keep it as it is already done

**Comment by notimetoname** [comment]
> Sure.

@roman-khimov added a commit that adopts `object/slicer`. `Relations` are broken (https://github.com/nspcc-dev/neofs-sdk-go/issues/416). RC8 is broken also, so added two commits that are kludges in fact.

Updating led to an uncontrollable refactor so let's stop somewhere here. Now it removes more than it adds, which is good. It does not look how i would keep it (object svc can be less complex), which is bad. After all, i do not want it to be more than just unnecessary changes.

File: cmd/neofs-node/container.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (roman-khimov) [reply cmd/neofs-node/container.go:403]: IIUC the idea behind this is to rotate nodes depending on the epoch, right? This certainly can be done without placement function, but we can't change the behavior this easy here. Maybe of course it's completely broken and thus irrelevant (see #1506), but until it's proven to be so I'd replicate the behavior we have in 0.36.1 (resorting to HRW).

**Review by roman-khimov — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**